canonical/landscape-client
# Fix Livepatch status reporting bug

Fix issue in which `pro status` inaccurately displays Livepatch status as "disabled" even if it is active.  Bug is due to the `landscape` user not being able to run snaps properly since it has a `$HOME` directory outside of `/home` (it's in `/var/lib/`).  

See https://bugs.launchpad.net/landscape-client/+bug/2037670

Manual Testing:
- Spin up a VM with a Livepatch-supported kernel
- Modify `UbuntuProInfo.run_interval` to be something quick like 10
- Modify `landscape-client.conf to point to a Landscape Server instance.  Additionally include `exchange_interval = 10` to exchange messages with server more quickly
- sudo scripts/landscape-monitor --config landscape-client.conf 
- Toggle livepatch and ensure updates are reflected in Server `sudo pro <enable/disable> livepatch`
